MODULE.bazel

```diff
@@ -13,6 +13,20 @@ bazel_dep(name = "rules_proto", version = "5.3.0-21.7")
 bazel_dep(name = "protobuf", version = "21.7", repo_name = "com_google_protobuf")
 
 bazel_dep(name = "stardoc", version = "0.6.2", dev_dependency = True, repo_name = "io_bazel_stardoc")
+bazel_dep(name = "rules_bazel_integration_test", version = "0.20.0", dev_dependency = True)
+
+# TODO(philsc): Can we load SUPPORTED_BAZEL_VERSIONS from version.bzl somehow?
+bazel_binaries = use_extension("@rules_bazel_integration_test//:extensions.bzl", "bazel_binaries", dev_dependency = True)
+
+# We no longer support Bazel 5.
```

Comment:
Yeah, I think it's fine to drop bazel 5 for the integration tests. Bazel 7 should be coming out soon, so I we're not missing much test coverage for long. You might have to update the presubmit.yml config to exclude stuff

Reply:
Okay, now at bazel 6.2 and 7.